conf.py

```diff
@@ -74,8 +75,17 @@
 # built documents.
 #
 # The short X.Y version.
-version = form_ltd_edition_name(
-    git_ref_name=os.getenv('TRAVIS_BRANCH', default='master'))
+github_ref = os.getenv("GITHUB_REF", "")
+if github_ref == "":
+    git_ref = "master"
+else:
+    match = re.match(r"refs/(heads|tags|pull)/(?P<ref>.+)", github_ref)
```

Comment:
We shouldn't actually be publishing from pull requests, though, given the condition in the workflow.